system_tests/program_test.go

```diff
@@ -84,6 +84,16 @@ func keccakTest(t *testing.T, jit bool) {
 	if otherVersion != programVersion {
 		Fatal(t, "mismatched versions", stylusVersion, programVersion)
 	}
+	programSize, err := arbWasm.ProgramSize(nil, programAddress)
+	Require(t, err)
+	if programSize < 5000 || programSize > 20000 {
+		Fatal(t, "unexpected size", programSize)
+	}
```

Comment:
I think we know the exact size and could check against it

Reply:
:+1: